app/views/inthenewspoets/items/_search_res_item.html.erb

```diff
@@ -1,4 +1,7 @@
 <%= search_result_item_field(locals[:item], "keywords", "Education") %>
 <%# note: below field needs to be parsed further %>
-<%= search_result_item_field(locals[:item], "subjects", "News Items") %>
-
+<%#= search_result_item_field(locals[:item], "subjects", "News Items") %>
+<% if locals[:item]["subjects"] %>
+  <% count = locals[:item]["subjects"].count %>
+  News Items: <%= count %>
+<% end %>
```

Comment:
Please delete the commented line.

Also, that partial wrote a little more HTML. If we're replacing it we probably want to keep similar HTML for any CSS that might be applied so it's consistent with the other fields' HTML: https://github.com/CDRH/african_poetics/blob/main/app/views/partials/_search_res_item_field.html.erb

Reply:
done